bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -834,6 +854,8 @@ protected void release(boolean wait, boolean writer)
         } else {
           future = null;
         }
+      } else if (storage == null) {
+        resumeSuspendedClients(-1);
```

Comment:
When spooling is disabled, it will wake up any suspended publishers that are waiting for the subscribers to catchup. It can be handled uniformly with canEvict, I will change it.